components/shared/TabItem.vue

```diff
@@ -6,8 +6,10 @@
       :no-shadow="noShadow"
       :active="active"
       :to="to"
-      :icon="active ? 'check' : ''">
```

Comment:
Have you checked if it would break something?  Several places use it.
<img width="339" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/854ba5c5-92ff-46a9-8bf7-4c4c27f930a7">

Reply:
In these cases only the label is shown. Should I add the check icon back?

Reply:
It should leave the check icon when only two options exist according to the requirement. The point is that you should not break the UI on the other components.

<img width="916" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/531685cd-35c1-49a7-b863-537ded0e5922">

Reply:
What about this `:icon="active && !slots.default ? 'check' : ''"`?
Or should I pass a `:show-icon="numberOfTabs == 2"` prop and  `:icon="active && showIcon ? 'check' : ''"` ?

Reply:
And also the should the Activity tab show a count also? In the design there is a History tab and it has no number.

Reply:
> Or should I pass a :show-icon="numberOfTabs == 2" prop and :icon="active && showIcon ? 'check' : ''"

This one